packages/terra-core-docs/src/terra-dev-site/doc/alert/AccessibilityGuide.2.doc.mdx

```diff
@@ -42,38 +43,103 @@ Terra Alert has an optional role prop but should be used carefully when the defa
+        It is recommended to separate them from less critical notification banners to prevent assistive technologies from confusing them.
+    </li>
+    <li>
+        By default, Terra Alert uses the "alert" notification type.
+    </li>
+</ul>
+
+Demo of Terra Alert with an action element.
+
+<AlertActionFocusDemo />
+
 #### Assistive technologies
 
 In combination with the notification criticality, screen reader users should understand the importance of the notification when presented.
 
 * The `"alert"` role will interrupt the user and announce the high critical notification immediately.
 * The `"status"` role will not interrupt the user and present politely.
 
-<Notice variant="important" ariaLevel="5">
+#### Live regions
+
+<ul>
+    <li>
+        For less critical notifications (non-alerts), the notification banner should be in an aria-live "polite" region (aria-live="polite").
+    </li>
+    <ul>
+        <li>
+            For screen readers to pick up and announce a newly rendered notification, the notification banner must be within an aria-live region.
+To enable assistive technologies the ability to inform the user of a less critical notification in a polite (non-interruptive) way, ensure the notification banner is
+within an element where aria-live="polite".
+        </li>
+    </ul>
+    <li>
+        Avoid mixing alert notifications and less critical notifications in the same region. See <a href="#accessibility-guidance-alert-notification-type">alert notification type guidance</a>.
+    </li>
+    <ul>
+        <li>
+            The "alert" notification type/role implicitly has aria-live value of `assertive` and will interrupt without being in a region where aria-live was previously set.
+It is recommended to separate critical alert notifications from less critical notification banners to prevent any confusion.
+        </li>
+    </ul>
+</ul>
+
+<div aria-label="Example code">
+
+    import Alert from 'terra-alert';
 
-###### Accessibility Guidance: Live regions
+    <div aria-live="polite">
+        {isOpen && (
+            <Alert type="success">
+                Meaningful alert content.
+            </Alert>
+        )}
+    </div>
+
+</div>
 
-For screen readers to pick up and announce a newly rendered notification, the notification must be within an aria-live region.
-To enable assistive technologies the ability to inform the user of a less critical notification in a polite (non-interruptive) way, ensure the Alert is
-within an element where `aria-live="polite"`.
+#### Titles
 
-Please note: the `"alert"` role implicitly has aria-live value of `assertive` and will interrupt without being in a region where aria-live is previously present.
+<ul>
+    <li>
+        Custom notification banners should have a title provided to the title prop.
+    </li>
+    <ul>
+        <li>
+            Titles are highly recommended for custom notification banners to have a more accessible notification.
```

Comment:
I feel like this can be combined with the list-item above, maybe saying "Custom notification banners should have a title provided to the title prop **in order to be accessible.**" (or something similar)

Reply:
Combined here https://github.com/cerner/terra-core/pull/3815/commits/a11aabf2402e790b9b84d306ed15481dea26d354#diff-2c4657d7282dff34f1995cecbdc9b487c519f018091c12370d911c9c60efd472R112